### PR TITLE
First commit of verifybamid

### PR DESCRIPTION
**Pull request by andreas-wilm**
* [X] I have read the guidelines above.
* [X] This PR adds a new recipe.
* [ ] This PR updates an existing recipe.
* [ ] This PR does something else (explain below).

@bioconda/core: Package building works. A local simulate-travis.py invocation runs until almost the end [failing with mulled-build](http://stackoverflow.com/questions/40745982/bioconda-no-such-file-or-directory-mulled-build), which I assume is just due to a not properly configured build environment.

### CODE REVIEW

File: recipes/verifybamid/build.sh [hunk 1]
```diff
@@ -0,0 +1,11 @@
+#!/bin/bash
+
+#strictly use anaconda build environment
+CC=${PREFIX}/bin/gcc
```
Comment (bgruening) [inline recipes/verifybamid/build.sh]: This should not be needed.

**Comment by bgruening** [comment]
Answered on Stackoverflow :)
This recipe looks really nice, you could use a template-var for you version number as well, but this is just nitpicking.

Thanks for your contribution!

**Comment by andreas-wilm** [comment]
Thanks! I don't need to make the suggested modifications and issue another pull request, right?

**Comment by bgruening** [comment]
You can simply add your changes to this PR :)

**Comment by andreas-wilm** [comment]
Even though you already made those changes above?

Stupid question: how do I add changes here? I thought there might be a corresponding edit or review button?

**Comment by bgruening** [comment]
There is a edit button if you look at the changes on the top right ob every file (box).
On a commandline you can simple commit and push to your branch.

**Comment by andreas-wilm** [comment]
Okay cool. Thanks for your help. 

Made the suggested changes and shamelessly cargo-culted stuff from the fraggenescan recipe to introduce template vars. Successfully Ran simulate-travis.py again and pushed to my branch.

Two more newbie questions:
1. Was I supposed to increase the build number?
1. Git is needed for a build. Do I need to mention this? 

Thanks,
Andreas

**Comment by bgruening** [comment]
> Two more newbie questions:
Was I supposed to increase the build number?

You only need to increase the build number if there is already a version deployed through bioconda. Otherwise, start with 0.

> Git is needed for a build. Do I need to mention this?

Where is this needed and why? If you really need it put `git` into your build dependencies.

Thanks,
Bjoern

**Comment by andreas-wilm** [comment]
Can't figure out from the travis logs what went wrong. All tests still complete successfully when run locally.

PS: Taking back what I said about git: it's only used in a different version of the package.

**Comment by daler** [comment]
Can you rebase and retry? We recently had build system issues that are now fixed (#3080)

**Comment by bgruening** [comment]
@andreas-wilm here is something wrong with the rebase. You have way to many files in this PR.

**Comment by andreas-wilm** [comment]
No idea what went wrong. I pulled from upstream and rebased. That was the idea, right? Would it be better if I close this one and start from scratch?

**Comment by bgruening** [comment]
As you like. 
If you change to master and than pull from upstream, go back to this branch and rebase against your master, it should work.

**Comment by andreas-wilm** [comment]
I somehow borked the rebase. Will close this pull request and start from scratch. Sorry

**Comment by andreas-wilm** [comment]
See https://github.com/bioconda/bioconda-recipes/pull/3105

**Pull request closed without merging**